app/controllers/api/v1/tags_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Api
+  module V1
+    class TagsController < ApplicationController
+      include Api::V1::Mixins::IndexMixin
+
+      def index
+        if params[:portfolio_id]
+          collection(Portfolio.find(params.require(:portfolio_id)).tags)
+        elsif params[:portfolio_item_id]
+          collection(PortfolioItem.find(params.require(:portfolio_item_id)).tags)
+        else
+          collection(Tag.all)
+        end
+      end
+
+      def show
+        tag = Tag.find(params.require(:id))
+
+        render :json => tag
+      end
```

Comment:
@lindgrenj6 Is it intentional that we won't have the API support adding/editing/deleting tags from API?

Reply:
Yes, that was not in the scope of the ticket. I was told we were ok with having to add tags manually (for now).